crates/starknet-types-rpc/src/generated/v0_5_0.rs

```diff
@@ -0,0 +1,1476 @@
+    Pending(PendingBlockWithTxHashes),
+}
+
+// #/methods/starknet_getBlockWithTxs/result/_anon
+#[derive(Serialize, Deserialize, Clone, Debug)]
+#[serde(untagged)]
+pub enum MaybePendingBlockWithTxs {
+    // #/methods/starknet_getBlockWithTxs/result/_anon/variant0
+    Block(BlockWithTxs),
+    // #/methods/starknet_getBlockWithTxs/result/_anon/variant1
+    Pending(PendingBlockWithTxs),
+}
+
+// #/methods/starknet_getClass/result/_anon
+#[derive(Serialize, Deserialize, Clone, Debug)]
+#[serde(untagged)]
+pub enum MaybeDeprecatedContractClass {
+    // #/methods/starknet_getClass/result/_anon/variant0
+    Deprecated(DeprecatedContractClass),
+    // #/methods/starknet_getClass/result/_anon/variant1
+    ContractClass(ContractClass),
+}
+
+// #/methods/starknet_getEvents/params/filter/_anon
+#[derive(Debug, Clone, Serialize, Deserialize)]
+pub struct EventFilterWithPageRequest {
+    // #/components/schemas/EVENT_FILTER/address
+    #[serde(default)]
+    pub address: Option<Address>,
+    // #/components/schemas/EVENT_FILTER/from_block
+    #[serde(default)]
+    pub from_block: Option<BlockId>,
+    // #/components/schemas/EVENT_FILTER/keys
+    /// The values used to filter the events
+    #[serde(default)]
+    pub keys: Option<Vec<Vec<Felt>>>,
+    // #/components/schemas/EVENT_FILTER/to_block
+    #[serde(default)]
+    pub to_block: Option<BlockId>,
+    // #/components/schemas/RESULT_PAGE_REQUEST/chunk_size
+    pub chunk_size: u64,
+    // #/components/schemas/RESULT_PAGE_REQUEST/continuation_token
+    /// The token returned from the previous query. If no token is provided the first page is returned.
+    #[serde(default)]
+    pub continuation_token: Option<String>,
+}
+
+// #/methods/starknet_getStateUpdate/result/_anon
+#[derive(Serialize, Deserialize, Clone, Debug)]
+#[serde(untagged)]
+pub enum MaybePendingStateUpdate {
+    // #/methods/starknet_getStateUpdate/result/_anon/variant0
+    Block(StateUpdate),
+    // #/methods/starknet_getStateUpdate/result/_anon/variant1
+    Pending(PendingStateUpdate),
+}
+
+// #/methods/starknet_getTransactionReceipt/result/_anon
+#[derive(Serialize, Deserialize, Clone, Debug)]
+#[serde(untagged)]
+pub enum MaybePendingTxnReceipt {
+    // #/methods/starknet_getTransactionReceipt/result/_anon/variant0
+    Receipt(TxnReceipt),
+    // #/methods/starknet_getTransactionReceipt/result/_anon/variant1
+    Pending(PendingTxnReceipt),
+}
+
+// #/methods/starknet_getTransactionStatus/result/_anon
+#[derive(Debug, Clone, Serialize, Deserialize)]
+pub struct TxnFinalityAndExecutionStatus {
+    // #/methods/starknet_getTransactionStatus/result/_anon/execution_status
+    #[serde(default)]
+    pub execution_status: Option<TxnExecutionStatus>,
+    // #/methods/starknet_getTransactionStatus/result/_anon/finality_status
+    pub finality_status: TxnStatus,
+}
+
+/// Parameters of the `starknet_specVersion` method.
+#[derive(Debug, Clone, Serialize, Deserialize)]
+pub struct SpecVersionParams {}
```

Comment:
Why is this one empty?

Reply:
The `starknet_specVersion` RPC call takes no parameters.